src/libmongoc/tests/test-mongoc-cache.c

```diff
@@ -21,9 +21,11 @@
 #include <mongoc/mongoc.h>
 #include <stdio.h>
 #include <signal.h>
-#include <bits/signum.h>
 #include "TestSuite.h"
 
+#ifndef SIGSTOP
+#define SIGSTOP 19
+
```

Comment:
```suggestion
#endif
```
that's what I did on alpine 3 12 and it works.

Reply:
Doh, thanks!